tests/Tests/Isolated/Common/Twig/fixtures/render/load-codes-no-rxcui.html

```
@@ -0,0 +1,85 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)